react/src/tests/opReturn.test.ts

```diff
@@ -3,33 +3,38 @@ import { exportedForTesting, parseOpReturnProps } from '../util/opReturn'
 const { stringToHex, prependPaymentIdWithPushdata, generatePushdataPrefixedPaymentId, getDataPushdata } = exportedForTesting
 
 describe('stringToHex', () => {
-  it ('Simple conversion 1', () => {
+  it ('Converts paybutton protocol lokad', () => {
     expect(stringToHex('PAY\x00')).toBe('50415900')
   })
-  it ('Simple conversion 2', () => {
+  it ('Converts ASCII printable and non printable', () => {
     expect(stringToHex('aA1=-*&]]^!\x04\x14')).toBe('6141313d2d2a265d5d5e210414')
   })
-  it('Works for non ASCII chars', () => {
+  it('Converts mix of ASCII, extended ASCII and non-ASCII', () => {
     expect(stringToHex('a©æ😂👌')).toBe('61c2a9c3a6f09f9882f09f918c')
   })
 });
 
 describe('prependPaymentIdWithPushdata', () => {
-  it ('Simple paymentId prepend 1', () => {
+  it ('Prepends three bytes', () => {
     expect(prependPaymentIdWithPushdata('aa00bb')).toBe('03aa00bb')
   })
-  it ('Simple paymentId prepend 2', () => {
+  it ('Prepends 45 bytes', () => {
```

Comment:
Actually I'd like to see a comment or something above to try explaining the "why" on these (including the ones below) as 3, 45, 75, etc all magic numbers to me. I see the 75 one explained lower down but not sure about the others.